exporter/lokiexporter/README.md

```diff
@@ -4,11 +4,12 @@
 | Status        |           |
 | ------------- |-----------|
 | Stability     | [beta]: logs   |
-| Distributions | [contrib], [observiq] |
+| Distributions | [contrib], [grafana], [observiq] |
```

Comment:
i couldn't find an import of the lokiexporter in the grafana-agent repo, is it there just not with the exporter module?

Reply:
Great catch, you are absolutely right! The documentation page says:

> NOTE: otelcol.exporter.loki is a custom component unrelated to the lokiexporter from the OpenTelemetry Collector.

Source: https://grafana.com/docs/agent/latest/flow/reference/components/otelcol.exporter.loki/